Windows: `cargo install television@0.4.18` failing

**Description**
Attempting to use:
```shell
cargo install television@0.4.18
```
is failing with:
```shell
error[E0433]: failed to resolve: use of undeclared crate or module `log`
  --> D:\.cache\cargo\registry\src\index.crates.io-6f17d22bba15001f\television-utils-0.0.0\src\stdin.rs:35:17
   |
35 |                 log::debug!(
   |                 ^^^ use of undeclared crate or module `log`

error[E0433]: failed to resolve: use of undeclared crate or module `log`
  --> D:\.cache\cargo\registry\src\index.crates.io-6f17d22bba15001f\television-utils-0.0.0\src\stdin.rs:46:9
   |
46 |         log::debug!(
   |         ^^^ use of undeclared crate or module `log`

error[E0433]: failed to resolve: use of undeclared crate or module `winapi_util`
  --> D:\.cache\cargo\registry\src\index.crates.io-6f17d22bba15001f\television-utils-0.0.0\src\stdin.rs:31:21
   |
31 |         let stdin = winapi_util::HandleRef::stdin();
   |                     ^^^^^^^^^^^ use of undeclared crate or module `winapi_util`

error[E0433]: failed to resolve: use of undeclared crate or module `winapi_util`
  --> D:\.cache\cargo\registry\src\index.crates.io-6f17d22bba15001f\television-utils-0.0.0\src\stdin.rs:32:25
   |
32 |         let typ = match winapi_util::file::typ(stdin) {
   |                         ^^^^^^^^^^^ use of undeclared crate or module `winapi_util`

For more information about this error, try `rustc --explain E0433`.
error: could not compile `television-utils` (lib) due to 4 previous errors
```

I cloned the repo to build from source and `main(0c13626d4c1b)` built fine.

**Environment**
 - OS: Windows
 - Rust version: 1.82.0
 - Project version: 0.4.18

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "television"
-version = "0.4.18"
+version = "0.4.19"
 edition = "2021"
 description = "The revolution will be televised."
 license = "MIT"
@@ -25,10 +25,10 @@ name = "tv"
 members = ["crates/television_channels","crates/television_derive", "crates/television_fuzzy", "crates/television_previewers", "crates/television_utils"]
 
 [dependencies]
-television-fuzzy = { version = "0.0.0", path = "crates/television_fuzzy" }
+television-fuzzy = { version = "0.0.1", path = "crates/television_fuzzy" }
 television-derive = { version = "0.0.0", path = "crates/television_derive" }
-television-channels = { version = "0.0.0", path = "crates/television_channels" }
-television-previewers = { version = "0.0.0", path = "crates/television_previewers" }
+television-channels = { version = "0.0.1", path = "crates/television_channels" }
+television-previewers = { version = "0.0.1", path = "crates/television_previewers" }
 television-utils = { version = "0.0.1", path = "crates/television_utils" }
 better-panic = "0.3.0"
 clap = { version = "4.4.5", features = [

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -29,7 +29,7 @@ television-fuzzy = { version = "0.0.0", path = "crates/television_fuzzy" }
 television-derive = { version = "0.0.0", path = "crates/television_derive" }
 television-channels = { version = "0.0.0", path = "crates/television_channels" }
 television-previewers = { version = "0.0.0", path = "crates/television_previewers" }
-television-utils = { version = "0.0.0", path = "crates/television_utils" }
+television-utils = { version = "0.0.1", path = "crates/television_utils" }
 better-panic = "0.3.0"
 clap = { version = "4.4.5", features = [
   "derive",

--- a/crates/television_channels/Cargo.toml
+++ b/crates/television_channels/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "television-channels"
-version = "0.0.0"
+version = "0.0.1"
 edition = "2021"
 description = "The revolution will be televised."
 license = "MIT"
@@ -15,7 +15,7 @@ categories = [
 ]
 
 [dependencies]
-television-fuzzy = { path = "../television_fuzzy", version = "0.0.0" }
+television-fuzzy = { path = "../television_fuzzy", version = "0.0.1" }
 television-utils = { path = "../television_utils", version = "0.0.1" }
 television-derive = { path = "../television_derive", version = "0.0.0" }
 devicons = "0.6.11"

--- a/crates/television_channels/Cargo.toml
+++ b/crates/television_channels/Cargo.toml
@@ -16,7 +16,7 @@ categories = [
 
 [dependencies]
 television-fuzzy = { path = "../television_fuzzy", version = "0.0.0" }
-television-utils = { path = "../television_utils", version = "0.0.0" }
+television-utils = { path = "../television_utils", version = "0.0.1" }
 television-derive = { path = "../television_derive", version = "0.0.0" }
 devicons = "0.6.11"
 tracing = "0.1.40"

--- a/crates/television_fuzzy/Cargo.toml
+++ b/crates/television_fuzzy/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "television-fuzzy"
-version = "0.0.0"
+version = "0.0.1"
 edition = "2021"
 description = "The revolution will be televised."
 license = "MIT"

--- a/crates/television_previewers/Cargo.toml
+++ b/crates/television_previewers/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "television-previewers"
-version = "0.0.0"
+version = "0.0.1"
 edition = "2021"
 description = "The revolution will be televised."
 license = "MIT"
@@ -16,7 +16,7 @@ categories = [
 
 [dependencies]
 syntect = "5.2.0"
-television-channels = { version = "0.0.0", path = "../television_channels" }
+television-channels = { version = "0.0.1", path = "../television_channels" }
 television-utils = { version = "0.0.1", path = "../television_utils" }
 tracing = "0.1.40"
 parking_lot = "0.12.3"

--- a/crates/television_previewers/Cargo.toml
+++ b/crates/television_previewers/Cargo.toml
@@ -17,7 +17,7 @@ categories = [
 [dependencies]
 syntect = "5.2.0"
 television-channels = { version = "0.0.0", path = "../television_channels" }
-television-utils = { version = "0.0.0", path = "../television_utils" }
+television-utils = { version = "0.0.1", path = "../television_utils" }
 tracing = "0.1.40"
 parking_lot = "0.12.3"
 tokio = "1.41.1"

--- a/crates/television_utils/Cargo.toml
+++ b/crates/television_utils/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "television-utils"
-version = "0.0.0"
+version = "0.0.1"
 edition = "2021"
 description = "The revolution will be televised."
 license = "MIT"

